### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by arniu**
## Motivation

Link to PRs

## Solution

Change `#3336` to `[#3336]`.

### CODE REVIEW

File: tokio/CHANGELOG.md [hunk 1]
```diff
@@ -3,107 +3,107 @@
 Commit to the API and long-term support.
 
 ### Fixed
-- sync: spurious wakeup in `watch` (#3234).
+- sync: spurious wakeup in `watch` ([#3234]).
```
Comment (taiki-e) [inline tokio/CHANGELOG.md:6]: > ([#3234]).

This (and all other links) is displayed like `[#3234]` in markdown. You have to add links to issues/PRs.

File: tokio/CHANGELOG.md [hunk 2]
```diff
@@ -666,174 +666,3 @@ another. This changelog entry contains a highlight
 # 0.1.0 (February 07, 2018)
 
 * Initial crate released based on [RFC](https://github.com/tokio-rs/tokio-rfcs/pull/3).
-
-[#2375]: https://github.com/tokio-rs/tokio/pull/2375
```
Comment (taiki-e) [inline tokio/CHANGELOG.md:670]: Why did you remove these links? This breaks all existing links.

**Review by taiki-e — CHANGES_REQUESTED** [review]
Thanks for the PR.

This patch seems to add links that do not work and break existing links as well.
I recommend seeing how these actually look: https://github.com/arniu/tokio/blob/patch-1/tokio/CHANGELOG.md

File: tokio/CHANGELOG.md [hunk 3]
```diff
@@ -3,107 +3,188 @@
...
+[#3287]: https://github.com/tokio-rs/tokio/pull/3287
+[#3289]: https://github.com/tokio-rs/tokio/pull/3289
+[#3298]: https://github.com/tokio-rs/tokio/pull/3298
+[#3299]: https://github.com/tokio-rs/tokio/pull/3299
+[#3304]: https://github.com/tokio-rs/tokio/pull/3304
+[#3306]: https://github.com/tokio-rs/tokio/pull/3306
+[#3306]: https://github.com/tokio-rs/tokio/pull/3306
+[#3308]: https://github.com/tokio-rs/tokio/pull/3308
+[#3308]: https://github.com/tokio-rs/tokio/pull/3308
+[#3308]: https://github.com/tokio-rs/tokio/pull/3308
```
Comment (taiki-e) [inline tokio/CHANGELOG.md:64]:
```suggestion
```

File: tokio/CHANGELOG.md [hunk 4]
```diff
@@ -3,107 +3,188 @@
...
+- net: `TcpSocket::reuseaddr()` ([#3093]).
+- net: `TcpSocket::local_addr()` ([#3093]).
+- net: add pid to `UCred` ([#2633]).
+
+[#2633]: https://github.com/tokio-rs/tokio/pull/2633
+[#3074]: https://github.com/tokio-rs/tokio/pull/3074
+[#3083]: https://github.com/tokio-rs/tokio/pull/3083
+[#3089]: https://github.com/tokio-rs/tokio/pull/3089
+[#3093]: https://github.com/tokio-rs/tokio/pull/3093
+[#3093]: https://github.com/tokio-rs/tokio/pull/3093
```
Comment (taiki-e) [inline tokio/CHANGELOG.md:164]:
```suggestion
```

File: tokio/CHANGELOG.md [hunk 5]
```diff
@@ -3,107 +3,188 @@
...
-- net: `UdpSocket::take_error()` (#3051).
-- sync: `oneshot::Sender::poll_closed()` (#3032).
+
+- io: `AsyncFd` for receiving readiness events on raw FDs ([#2903]).
+- net: `poll_*` function on `UdpSocket` ([#2981]).
+- net: `UdpSocket::take_error()` ([#3051]).
+- sync: `oneshot::Sender::poll_closed()` ([#3032]).
+
+[#2903]: https://github.com/tokio-rs/tokio/pull/2903
+[#2903]: https://github.com/tokio-rs/tokio/pull/2903
```
Comment (taiki-e) [inline tokio/CHANGELOG.md:183]:
```suggestion
```

File: tokio/CHANGELOG.md [hunk 6]
```diff
@@ -140,136 +230,266 @@ Biggest changes are:
...
+[#2845]: https://github.com/tokio-rs/tokio/pull/2845
+[#2856]: https://github.com/tokio-rs/tokio/pull/2856
+[#2861]: https://github.com/tokio-rs/tokio/pull/2861
+[#2876]: https://github.com/tokio-rs/tokio/pull/2876
+[#2882]: https://github.com/tokio-rs/tokio/pull/2882
+[#2885]: https://github.com/tokio-rs/tokio/pull/2885
+[#2890]: https://github.com/tokio-rs/tokio/pull/2890
+[#2893]: https://github.com/tokio-rs/tokio/pull/2893
+[#2893]: https://github.com/tokio-rs/tokio/pull/2893
+[#2893]: https://github.com/tokio-rs/tokio/pull/2893
```
Comment (taiki-e) [inline tokio/CHANGELOG.md:318]:
```suggestion
```

File: tokio/CHANGELOG.md [hunk 7]
```diff
@@ -385,22 +674,47 @@ Biggest changes are:
 - `StreamExt::merge` combines two streams, yielding values as they become ready ([#2091]).
 - Task-local storage ([#2126]).
 
+[#1762]: https://github.com/tokio-rs/tokio/pull/1762
+[#2030]: https://github.com/tokio-rs/tokio/pull/2030
+[#2085]: https://github.com/tokio-rs/tokio/pull/2085
+[#2089]: https://github.com/tokio-rs/tokio/pull/2089
+[#2091]: https://github.com/tokio-rs/tokio/pull/2091
+[#2092]: https://github.com/tokio-rs/tokio/pull/2092
+[#2092]: https://github.com/tokio-rs/tokio/pull/2092
```
Comment (taiki-e) [inline tokio/CHANGELOG.md:683]:
```suggestion
```

File: tokio/CHANGELOG.md [hunk 8]
```diff
@@ -425,14 +740,38 @@ Biggest changes are:
 - `time::DelayQueue::len` returns the number entries in the queue ([#1755]).
 - expose runtime options from the `#[tokio::main]` and `#[tokio::test]` ([#2022]).
 
+[#1699]: https://github.com/tokio-rs/tokio/pull/1699
+[#1755]: https://github.com/tokio-rs/tokio/pull/1755
+[#1870]: https://github.com/tokio-rs/tokio/pull/1870
+[#1971]: https://github.com/tokio-rs/tokio/pull/1971
+[#2001]: https://github.com/tokio-rs/tokio/pull/2001
+[#2001]: https://github.com/tokio-rs/tokio/pull/2001
```
Comment (taiki-e) [inline tokio/CHANGELOG.md:748]:
```suggestion
```

File: tokio/CHANGELOG.md [hunk 9]
```diff
@@ -444,24 +783,48 @@ Biggest changes are:
...
+
 - runtime threads are configured with `runtime::Builder::core_threads` and
   `runtime::Builder::max_threads`. `runtime::Builder::num_threads` is
   deprecated ([#1977]).
 
+[#1924]: https://github.com/tokio-rs/tokio/pull/1924
+[#1933]: https://github.com/tokio-rs/tokio/pull/1933
+[#1934]: https://github.com/tokio-rs/tokio/pull/1934
+[#1939]: https://github.com/tokio-rs/tokio/pull/1939
+[#1939]: https://github.com/tokio-rs/tokio/pull/1939
```
Comment (taiki-e) [inline tokio/CHANGELOG.md:802]:
```suggestion
```

File: tokio/CHANGELOG.md [hunk 10]
```diff
@@ -473,14 +836,33 @@ Biggest changes are:
...
-- improved documentation (#1865, #1866, #1868, #1874, #1876, #1911).
+- improved documentation ([#1865], [#1866], [#1868], [#1874], [#1876], [#1911]).
+
+[#1863]: https://github.com/tokio-rs/tokio/pull/1863
+[#1864]: https://github.com/tokio-rs/tokio/pull/1864
+[#1865]: https://github.com/tokio-rs/tokio/pull/1865
+[#1866]: https://github.com/tokio-rs/tokio/pull/1866
+[#1868]: https://github.com/tokio-rs/tokio/pull/1868
+[#1874]: https://github.com/tokio-rs/tokio/pull/1874
+[#1874]: https://github.com/tokio-rs/tokio/pull/1874
```
Comment (taiki-e) [inline tokio/CHANGELOG.md:851]:
```suggestion
```

File: tokio/CHANGELOG.md [hunk 11]
```diff
@@ -490,27 +872,48 @@ Biggest changes are:
...
 - `File::into_std` and `File::try_into_std` methods ([#1856]).
 
+[#1772]: https://github.com/tokio-rs/tokio/pull/1772
+[#1834]: https://github.com/tokio-rs/tokio/pull/1834
+[#1839]: https://github.com/tokio-rs/tokio/pull/1839
+[#1841]: https://github.com/tokio-rs/tokio/pull/1841
+[#1843]: https://github.com/tokio-rs/tokio/pull/1843
+[#1849]: https://github.com/tokio-rs/tokio/pull/1849
+[#1854]: https://github.com/tokio-rs/tokio/pull/1854
+[#1854]: https://github.com/tokio-rs/tokio/pull/1854
```
Comment (taiki-e) [inline tokio/CHANGELOG.md:888]:
```suggestion
```

File: tokio/CHANGELOG.md [hunk 12]
```diff
@@ -522,318 +925,230 @@ another. This changelog entry contains a highlight
...
+- Introduce Tokio Runtime ([#141])
+- Provide `CurrentThread` for more flexible usage of current thread executor ([#141]).
+- Add Lio for platforms that support it ([#142]).
+- I/O resources now lazily bind to the reactor ([#160]).
+- Extract Reactor to dedicated crate ([#169])
+- Add facade to sub crates and add prelude ([#166]).
+- Switch TCP/UDP fns to poll\_ -> Poll<...> style ([#175])
+
+[#141]: https://github.com/tokio-rs/tokio/pull/141
+[#141]: https://github.com/tokio-rs/tokio/pull/141
```
Comment (taiki-e) [inline tokio/CHANGELOG.md:1141]:
```suggestion
```

Comment (arniu) [reply tokio/CHANGELOG.md:1141]: I'll run dedup on it.

**Review by Darksonn — APPROVED** [review]
Can you include the 1.0.1 release as well?

**Review by Darksonn — APPROVED** [review]

**Review by taiki-e — APPROVED** [review]

**Pull request merged**